### PR TITLE
Highlight data classes

### LINKED ISSUE #6: Anthropomorphic, humanoid and testbed file class
from Nevio:

> I am not sure if this file, the "Humanoid anthropometric measures" file and the "Testbed-related data" file should be included in the "Pre-Processed Data Files" class or in a different class of files.

### PR DESCRIPTION
**Pull request by aremazeilles**
Fix #6 

I restructured the file to highlight the data types, as commented in #6.
I added a 4th class, for raw data file.

Some additional points to mention:

* the bipedal system section is unclear to me. I gathered all subsections related to it, and it does not seem to be completely consistent. Would need revision. I think we have several issues related to that aspect already: #5, #20, #21
* I simplified the testbed naming aspects. For me, a protocol involves one setting file, whatever is the equipment involved. So there is no need reminding the protocol name in the testbed name.
* If a testbed can change along time (like changing the slope), I suggest adding a number to identify it. We should also refer to that number in other recorded file.  Other issue?

### CODE REVIEW

File: data_format.adoc [hunk 1]
```diff
@@ -113,10 +116,100 @@ From this file, the number of joints, its labels and the degrees of freedom can
...
+All these segments are linked by joints (fixed, prismatic, rotational) forming a single tree.
+Moreover, it allows to use a wide variety of simulators commonly used in robotics such as Gazebo.
+
+**Number of files**: Usually each segment, sensor, or set of segments such as a leg are described in a single file. Finally the whole robot includes all these files in a single file which is the one loaded.
+
+**File Format**: .urdf. This format file allows to include Gazebo simulation tags, such as friction properties, or visualization properties that allow to simulate more realistic behaviours.
+This file shall contain the dimensions and inertial properties of each segment of the worn robot with respect to the reference system of the human body segment connected to it.
+This is needed to enable dynamic simulators to model the human-WR system.
+
+**name of the file**: `wr_N_config_M_segments`, where `N` is the WR number and `M` is the configuration number (for resizable robots this could be useful).
```
Comment (alfonsotecnalia) [inline data_format.adoc:136]: `name of the file` is before `file format` in the other types

File: data_format.adoc [hunk 2]
```diff
@@ -113,10 +116,100 @@ From this file, the number of joints, its labels and the degrees of freedom can
...
+
+**file format**: .yaml
```
Comment (alfonsotecnalia) [inline data_format.adoc:138]: Line 138 should be removed

File: data_format.adoc [hunk 3]
```diff
@@ -113,10 +116,100 @@ From this file, the number of joints, its labels and the degrees of freedom can
 
 **Number of files**: all necessary files to describe the complete robotic structure.
 
-**File format**: .urdf.
-The use of .urdf files also has shortcomings such as the lack of friction (important for e.g. walking steeper slope angles).
+**File format**: `.urdf`.
```
Comment (alfonsotecnalia) [inline data_format.adoc:119]: Lack of `name of the file`

**Comment by aremazeilles** [comment]
Ready to merge @alfonsotecnalia ?

**Comment by alfonsotecnalia** [comment]
Done!

**Pull request merged**